pymbolic/primitives.py

```diff
@@ -1796,13 +1796,13 @@ def wrap_in_cse(expr, prefix=None):
         if prefix is None:
             return expr
         if expr.prefix is None and type(expr) is CommonSubexpression:
```

Comment:
Use same logic as `make_cse` below.

Reply:
Hm, I'm not sure what you had in mind here? At least at a first glance, these seem to have different logic: `make_cse` ignores the prefix when re-wrapping and `wrap_in_cse` ignores the scope when re-wrapping.

Reply:
To be honest, I'm not sure why we have both of these, and I think the differences are mostly spurious. 

The one actual difference seems to be that `Variable` and `Subscript` aren't re-wrapped, but that could be a flag on `make_common_subexpression`, at which point `wrap_in_cse` could be deprecated.

Reply:
`wrap_in_cse` seems to be exclusively used in `sumpy`, so I'm guessing it needed a fast re-wrap thing at some point.

I went ahead and deprecated it and replaced it with a flag in `make_common_subexpression` in
https://github.com/inducer/pymbolic/compare/3bcf915c6b2a360eb09b98c7aba4d2641cfccf4e..451d51fe3936d7d157de7ab766d7a0d1bda65518